src/tests/ftest/util/server_utils.py

```diff
@@ -17,6 +17,7 @@
 from command_utils_base import CommonConfig
 from exception_utils import CommandFailure
 from command_utils import SubprocessManager
+from command_utils_base import BasicParameter
```

Comment:
(pylint-ungrouped-imports) Imports from package command_utils_base are not grouped